lale/pretty_print.py

```diff
@@ -582,7 +580,7 @@ def _combine_lonely_literals(printed_code):
     regex = re.compile(
         r' +("[^"]*"|\d+\.?\d*|\[\]|float\("nan"\)|np\.dtype\("[^"]+"\)),'
     )
-    for i in range(len(lines)):
+    for i in range(len(lines)):  # pylint:disable=consider-using-enumerate
```

Comment:
Maybe enumerate would indeed make this code clearer?

Reply:
I am not sure, since the code also assigns to lines[i].  Given that, using `lines[i]` for both reading and writing offers some possible benefits to clarity.